src/community/taskmanager/pom.xml

```diff
@@ -28,6 +28,11 @@
       <version>1.8.5</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <version>1.1.119</version>
```

Comment:
Do we need the version number? Should be managed in the dependency management of the main pom.xml file of GeoServer

Reply:
You're right. I missed that community project is also a child of geoserver parent pom. Removed.